lib/extensions.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:tinkoff_acquiring_sdk/models/tinkoff_tax.dart';
+import 'package:tinkoff_acquiring_sdk/models/tinkoff_taxation.dart';
+
+extension TaxationToString on TinkoffTaxation {
```

Comment:
I think this should be generalised (will look into generifying enum in dart later)